docs/how-to/llm-fine-tuning-optimization/model-acceleration-libraries.rst

```diff
@@ -251,3 +251,265 @@ page describes the options.
+
+#. Install the GCC C/C++ compiler. You must install a version of the compiler that supports C++20. GCC also requires the ``sysroot`` package.
+
+   .. note::
+
+      Newer versions of GCC can be used, but they are not compatible with older
+      systems such as Ubuntu 20.04. This is because the compiled library could
+      reference symbols from unsupported versions of ``GLIBCXX``. For more information, see the
+      `PyTorch FBGEMM build tools documentation <https://pytorch.org/FBGEMM/fbgemm_gpu-development/BuildInstructions.html#install-the-build-tools>`_.
+
+   .. code-block:: shell
+
+      # Set GCC to 10.4.0 to keep compatibility with older versions of GLIBCXX
+      #
+      # A newer versions of GCC also works, but will need to be accompanied by an
+      # appropriate updated version of the sysroot_linux package.
+      gcc_version=10.4.0
+
+      conda install -n ${env_name} -c conda-forge -y gxx_linux-64=${gcc_version} sysroot_linux-64=2.17
+
+#. Add symlinks for the C/C++ compiler to the binary path. This overwrites any existing symlinks. In a Miniconda environment, the 
+   binary path is located at ``$CONDA_PREFIX/bin``.
+
+   .. code-block:: shell
+
+      conda_prefix=$(conda run -n ${env_name} printenv CONDA_PREFIX)
+
+      ln -sf "${path_to_gcc}" "$(conda_prefix)/bin/cc"
+      ln -sf "${path_to_gcc}" "$(conda_prefix)/bin/c++"
+
+#. Install some additional build tools.
+
+   .. code-block:: shell
+
+      conda install -n ${env_name} -y \
+         click \
+         cmake \
+         hypothesis \
+         jinja2 \
+         make \
+         ncurses \
+         ninja \
+         numpy \
+         scikit-build \
+         wheel
+
+Install PyTorch
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Install `PyTorch <https://pytorch.org/>`_ using ``pip`` for the most reliable and consistent results.
+
+#. Install PyTorch using ``pip``.
+
+   .. code-block:: shell
+
+      # Install the latest nightly, ROCm variant
+      conda run -n ${env_name} pip install --pre torch --index-url https://download.pytorch.org/whl/nightly/rocm6.2/
+
+#. Ensure PyTorch loads correctly and verify the version and variant of the installation using an ``import`` test.
+
+   .. code-block:: shell
+
+      # Ensure that the package loads properly
+      conda run -n ${env_name} python -c "import torch.distributed"
+
+      # Verify the version and variant of the installation
+      conda run -n ${env_name} python -c "import torch; print(torch.__version__)"
+
+Perform the pre-build and build
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+#. Clone the FBGEMM repository and the relevant submodules. Use ``pip`` to install the 
+   components in ``requirements.txt``. Run the following commands inside the Miniconda environment.
+
+   .. code-block:: shell
+
+      # Select a version tag
+      FBGEMM_VERSION=v0.8.0
+
+      # Clone the repo along with its submodules
+      git clone --recursive -b ${FBGEMM_VERSION} https://github.com/pytorch/FBGEMM.git fbgemm_${FBGEMM_VERSION}
```

Comment:
Shall we use https://github.com/ROCm/FBGEMM.git or https://github.com/pytorch/FBGEMM.git ? @amathews-amd

Reply:
Hi @amd-jnovotny , please change the command to
`git clone https://github.com/pytorch/FBGEMM.git --branch=v0.8.0 --recursive fbgemm_${FBGEMM_VERSION}`